### DISCUSSION TITLE
downloading replays via website

### DISCUSSION THREAD

**Discussion by GavinMinus** [ppy/osu#18115] [Ideas]
I already know you can download replays on the website but I'm only available to download a select few, just wondering if you were able to make it so people were able to download any replay from their top 100. I just find it kind of weird I'm only able to download 5 or 6 replays from my top 100.

example of what I mean:
![image](https://user-images.githubusercontent.com/105038735/167078618-cdd0ee02-4b24-4428-8470-b886a471f127.png)
and also I can only download replays from old.ppy.sh idk why that is a thing.

**Comment by peppy** [comment]
Only replays which are quite high on a beatmap's leaderboard are available. If we stored all user replays it would take up terabytes and become unaffordable. We hope to store all replays for supporters in the future though.

As for downloading replays using the --- never mind.

**Comment by bdach** [comment]
Also you should be able to download replays on new website through this thing:

![image](https://user-images.githubusercontent.com/20418176/167079046-d109ae9a-7652-4266-bd5c-17552a13952d.png)
